frontend/test/lit/test_measurements.py

```diff
@@ -395,6 +395,65 @@ def var1(x: float, y: float):
 print(var1.mlir)
 
 
+# CHECK-LABEL: private @var2(
```

Comment:
I don't really think we need all these tests. Or at least we don't need to CHECK the `RX`, `RY`,... only the variance.

Reply:
I'd agree! Let me fix this.